packages/reference/src/components/MissingEntityCard/MissingEntityCard.tsx

```diff
@@ -1,37 +1,42 @@
-import * as React from 'react';
+import React from 'react';
 
-import { Card, Flex, IconButton, SectionHeading } from '@contentful/f36-components';
+import { EntryCard, IconButton } from '@contentful/f36-components';
 import { CloseIcon } from '@contentful/f36-icons';
 
-import { ContentEntityType } from '../../types';
-import * as styles from './styles';
-
 export function MissingEntityCard(props: {
-  entityType: ContentEntityType;
-  asSquare?: boolean;
+  isDisabled?: boolean;
   isSelected?: boolean;
-  isDisabled: boolean;
   onRemove?: Function;
+  providerName?: string;
 }) {
+  const providerName = props.providerName ?? 'Source';
+  const description =
+    props.providerName === 'Contentful'
+      ? "Content was deleted or archived or you don't have access rights."
+      : `Content was deleted or archived or you don't have access rights from the external provider. Contact your technical team.`;
```

Comment:
> Contact your technical team

Do you think it helps them to get more help?

Reply:
The goal is for them to reach out to their support and not us because we know the reason for the error is them and not us. Happy to hear any suggestions you have on how we can convey this message.

Reply:
the challenge I anticipate is actually finding someone who is in the position to help
1. How would anyone know to whom to reach out? In other words, who is the technical team for a content editor?
2. Would that be useful for a user of the Marketplace app (e.g. Shopify) that has an app installed on their space but does not manage the integration itself?

the alternative message could look like:
```
We were not able to find %YOUR_RESOURCE% with urn:%your_urn%. 

This may be due to various reasons beyond our control, such as changes or restrictions on the external provider's side. Verify the URN for accuracy.
```